test/truefi2/lines-of-credit/TrueRateAdjuster.test.ts

```diff
@@ -45,6 +45,7 @@ describe('TrueRateAdjuster', () => {
   let asset: MockErc20Token
   let mockSpotOracle: MockContract
   let oracle: TimeAveragedBaseRateOracle
+  let mockFactory: IPoolFactory
```

Comment:
I think importing `IPoolFactory` is not necessary. This variable can just be of type `MockPoolFactory`